README.md

```diff
@@ -123,10 +123,10 @@ any form of iterator, different iterators have to be handled differently.
 | starmap                     | ☐    | ☑      | ☐             | ☐  |
 | tee                         | ☐    | ☑      | ☐             | ☐  |
 | compact                     | ☐    | ☐      | ☑             | ☐  |
-| contains                    | ☐    | ☐      | ☑             | ☑  |
+| contains                    | ☐    | ☑      | ☑             | ☑  |
 | range                       | ☑    | ☑      | ☑             | ☑  |
 | reduce                      | ☑    | ☑      | ☑             | ☑  |
-| sorted                      | ☐    | ☐      | ☑             | ☐  |
+| sorted                      | ☐    | ☑      | ☑             | ☐  |
```

Comment:
`sorted` seems less meaningful in the context of iterator helpers as it cannot be lazy. `itertools.js` simply returns an `Array<T>`, so does Python's built-in `sorted`.